doc/developers/contributing.rst

```diff
@@ -44,6 +44,17 @@ ticket to the
 <https://github.com/scikit-learn/scikit-learn/issues>`_. You are also
 welcome to post feature requests or pull requests.
 
+Automated Contributions Policy
+==============================
+
+Please refrain from submitting mass issues or pull requests generated by
+fully-automated tools. Maintainers reserve the right to close issues and pull
+requests from fully-automated tools at their discretion. Persistent automated
+submissions may result in blocking the offending account.
```

Comment:
I would probably change this to "Maintainers might block any account submitting automated pull requests or issues at any time.", to be able to block immediately.